### ISSUE TITLE
filterExpression not correctly passed to Event

### ISSUE THREAD

**Issue by Mercwri** [halworsen/fflogsapi#2]
It looks like the filterExpression argument for events is not correctly passed to API resulting in no event data found.

For example, making a query on each fight in a given report, and setting the filterExpression argument results in either no data returned (example 1) or an API error (example 2).

Example 1:
![image](https://github.com/halworsen/fflogsapi/assets/8934100/9239d686-ff88-4e37-bcff-7d789dad6134)

Example 2:
![image](https://github.com/halworsen/fflogsapi/assets/8934100/ba7266a1-c937-42e9-98fb-55a42dd2c391)

**Comment by halworsen** [comment]
The filter expression in example 1 is malformed, the type needs to be surrounded in quotes. You can see this by using the filter in the report itself:
https://www.fflogs.com/reports/gqnCrN3B6yz8PH4A#fight=12&pins=2%24Off%24%23244F4B%24expression%24type%3Dlimitbreakupdate&view=events

The quotation bug was fixed quite a long time ago on the dev branch but has not been released yet as it was part of a phase information feature that is not fully completed: https://github.com/halworsen/fflogsapi/commit/dddb5aaeb20b52aa81b9b858d33cf0c92d2f605a

For the time being, you can manually insert escapes for the quotes:
```python
>>> fight.events(filters={'filterExpression': 'type=\\"targetabilityupdate\\"'})
[{'timestamp': 6690437, 'type': 'targetabilityupdate', 'sourceID': 10, 'targetID': 10, 'abilityGameID': 0, ...
```

**Comment by halworsen** [comment]
Fixed in [v2.1.0](https://github.com/halworsen/fflogsapi/releases/tag/v2.1.0)
